components/fxa-client/src/internal/commands/close_remote_tabs.rs

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use base64::{engine::general_purpose::URL_SAFE_NO_PAD, Engine};
+use rc_crypto::ece;
+use serde_derive::*;
+
+use crate::{Error, Result, ScopedKey};
+
+use super::{
+    super::device::Device,
+    send_tab::{PrivateSendTabKeysV1, PublicSendTabKeys, SendTabKeysPayload},
+};
+
+pub const COMMAND_NAME: &str = "https://identity.mozilla.com/cmd/close-uris";
```

Comment:
```suggestion
pub const COMMAND_NAME: &str = "https://identity.mozilla.com/cmd/close-uri";
```

https://phabricator.services.mozilla.com/D204556 calls this `close-uri`, and it also makes sense for it to be singular if we only send one closed tab URL per command.

Reply:
As above, it's not clear if we want to fully commit to this yet. In general, and IMO, the name should match the payload (ie, plural if the payload holds a vec, singular otherwise)

Reply:
The capability URL that Desktop uses is `https://identity.mozilla.com/cmd/close-uri/v1`, so I changed this one to match.